install/on-host-integration/tidbcloud/install.yml

```diff
@@ -0,0 +1,15 @@
+id: tidbcloud-integration-docs
```

Comment:
the id should probably be just 'tidbcloud' to match the rest

Reply:
done